pkg/api/v1beta1/types.go

```diff
@@ -474,6 +474,7 @@ type PodTemplate struct {
 	DesiredState PodState          `json:"desiredState,omitempty" description:"specification of the desired state of pods created from this template"`
 	NodeSelector map[string]string `json:"nodeSelector,omitempty" description:"a selector which must be true for the pod to fit on a node"`
 	Labels       map[string]string `json:"labels,omitempty" description:"map of string keys and values that can be used to organize and categorize the pods created from the template; must match the selector of the replication controller to which the template belongs; may match selectors of services"`
+	Annotations  map[string]string `json:"annotations,omitempty"`
```

Comment:
Good catch, you're right: I don't seem to have the installed. Will remedy that immediately. Thanks.